presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/ThriftModule.java

```diff
@@ -64,8 +64,7 @@ public void configure(Binder binder)
         configBinder(binder).bindConfig(ThriftConnectorConfig.class);
         binder.bind(ThriftSessionProperties.class).in(Scopes.SINGLETON);
         binder.bind(ThriftIndexProvider.class).in(Scopes.SINGLETON);
-        newExporter(binder).export(PrestoThriftServiceProvider.class)
-                .as(generatedNameOf(RetryingPrestoThriftServiceProvider.class, connectorId));
+        newExporter(binder).export(PrestoThriftServiceProvider.class).as(generatedNameOf(RetryingPrestoThriftServiceProvider.class, connectorId));
```

Comment:
Unrelated formatting change?

Reply:
BTW, the name `RetryingPrestoThriftServiceProvider` seems quite long. Should we shorten the exported name?

Reply:
The change is indeed unrelated, but I hesitated to put it in a separate commit. I removed it for now.